libs/core/src/decorators/README.md

```diff
@@ -0,0 +1,48 @@
+# Debounce Decorator
+
+A utility decorator that delays method execution until after a specified delay period has elapsed since it was last invoked.
+
+## Usage
+
+```ts
+import { debounce } from '@daffodil/core';
+
+class SearchComponent {
+  @debounce(300)
+  onSearchInput(value: string) {
+    // Only executes after 300ms
+    this.performSearch(value);
+  }
+}
+```
+
+## API
```

Comment:
@damienwebdev should there be a Utilities subsection in the `@daffodil/core` guides where this goes?